lib/sequenceserver.rb

```diff
@@ -318,12 +320,14 @@ def server_url
       "http://#{host}:#{config[:port]}"
     end
 
+    #Returns a local ip adress
     def ip_address
       Socket.ip_address_list.find { |ai| ai.ipv4? && !ai.ipv4_loopback? }.ip_address
     end
 
+    #Returns machine's hostname based on the local ip
```

Comment:
What does the method return if hostname could not be determined? That should be included in the documentation as well.